### ISSUE TITLE
Release 1.19 broke existing 1.18 installations

### ISSUE THREAD

**Issue by kmurphy4** [youtype/mypy_boto3_builder#114]
**Describe the bug**
I'm working on a project that's pinned to `boto3-stubs[kms]==1.18.40`.  It has been type-checking successfully for a few weeks now.  However, I went to install it on a new machine, and `mypy` complained about mismatched types.  Specifically, the errors were
```python
path/to/kms.py: note: In function "decrypt":                                                                  
path/to/kms.py:20:5: error: Returning Any from function declared to return "str"  [no-any-return]             
        return response["Plaintext"].decode("utf-8")                                                                                    
        ^                                                                                                                               
path/to/kms.py:20:12: error: "StreamingBody" has no attribute "decode"  [attr-defined] 
        return response["Plaintext"].decode("utf-8")                                                                                    
               ^                                                                                                                        
path/to/kms.py: note: In function "encrypt":                                                                  
path/to/kms.py:39:29: error: Argument 1 to "b64encode" has incompatible type "StreamingBody"; expected        
"bytes"  [arg-type]                                                                                                                     
        return base64.b64encode(ciphertext["CiphertextBlob"])                                                                           
                                ^                                                                                                       
Found 3 errors in 1 file (checked 25 source files)                                                                                      
```

I'm not really familiar with the actual errors (I haven't looked into them yet).

What does surprise me is that I seem to have the wrong version of `mypy-boto3-kms` installed (even though it was fully pinned, AFAICT).
```
$ pip freeze | grep boto3     
boto3==1.18.40                                                                               
boto3-stubs==1.18.40
mypy-boto3-kms==1.19.1
```

Looking into it a bit more, I downloaded the sdist from https://pypi.org/project/boto3-stubs/1.18.40/#files and looked at `setup.py`, which contains
```python
extras_require={
    "all": [
        ...,
        "mypy-boto3-kms>=1.18.35",
        ...,
    ]
}
```

Basically, I think the issue here is that you need `>=1.18.35,<1.19` instead of just `>=1.18.35`.  Looks like there should just be a  `,<{{ max_build_version }}` part here: https://github.com/vemel/mypy_boto3_builder/blob/9f2cf87bba255ac71199ef0260b4cbe14bbcedf2/mypy_boto3_builder/templates/boto3-stubs/setup.py.jinja2#L52

On my end, I can just pin it manually (so I'm not blocked), but I just thought this would be best fixed at the distribution layer.
